Exiled.API/Features/Badge.cs

```diff
@@ -30,21 +30,21 @@ public Badge(string text, string color, int type, bool isGlobal = false)
         /// <summary>
         /// Gets the badge text.
         /// </summary>
-        public string Text { get; private set; }
```

Comment:
I was thinking about giving the possibility to use the `Badge` struct in configs, we shouldn't remove setters and we should create a public default constructor, so YamlDotNet will be able to deserialize it.

Reply:
I've restored the private setters.